_layouts/partners/faq.html

```diff
@@ -0,0 +1,194 @@
+    }else{
+      openAllAccordions(section);
+    }
+    toggleExpandAllCollapseAllBtn(section);
+  }
+
+  function closeAllAccordions(section){
+    let accordionButtons = document.getElementsByClassName('login-accordion-btn-' + section)
+    
+    Array.from(accordionButtons).forEach(el => {
+      el.setAttribute("aria-expanded", "false");
+    });
+
+    let accordionContentPanels = document.getElementsByClassName('login-accordion-content-' + section);
+    
+    Array.from(accordionContentPanels).forEach(el => {
+      el.hidden = true;
+    })
+
+    openAccordionCount[section] = 0;
+  };
+  
+  function openAllAccordions(section){
+    let accordionButtons = document.getElementsByClassName('login-accordion-btn-' + section)
+    openAccordionCount[section] = 0;
+
+    Array.from(accordionButtons).forEach(el => {
+      el.setAttribute("aria-expanded", "true");
+      openAccordionCount[section] ++;
+    });
+    
+    let accordionContentPanels = document.getElementsByClassName('login-accordion-content-' + section);
+    Array.from(accordionContentPanels).forEach(el => {
+      el.hidden = false;
+    })
+  };
+</script>
+<main id="main-content">
+    <section>
+        <div class="container partners-container">
+            <div class="grid-row">
+                <h1 class="partners-title">{{ page.title }}</h1>
+                <div class="tablet:grid-col-9 accordion-container">
+                  <div class="partners-border"></div>
+                  <div class="section-title-row">
+                    <div id="general-section" class="section-title-left">{{ page.general_section_title | markdownify}}</div>
+                    <a 
+                      class="section-title-right collapse-expand-btn-general" 
+                      href="javascript:updateAllAccordions('general')">
+                      Expand all
+                    </a>
+                  </div>
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5" aria-multiselectable="true">
+                      {% for item in page.general_accordion %}
+                          <div class="usa-accordion__heading">
+                              <button
+                                onclick="updateSingleAccordion(this, 'general')"
+                                class="usa-accordion__button login-accordion-btn-general"
+                                aria-expanded="false"
+                                aria-controls={{ item.id }}
+                                >
+                                {{ item.title }}
+                              </button>
+                          </div>
+                          <div id={{ item.id }} class="usa-accordion__content login-accordion-content-general">
+                          {{ item.content | markdownify}}
+                          </div>
+                      {% endfor %}
+                  </div>
+                  <a href="#main-content" class="usa-current">Back to Top</a>
```

Comment:
Wondering if `usa-current` was there to add some coloring, since removing it causes browser default. I assume we'd want:

```suggestion
                  <a href="#main-content" class="usa-link">Back to Top</a>
```